app/models/meeting.js

```diff
@@ -1,96 +1,41 @@
-  startedOn: attr('datetime'),
-  endedOn: attr('datetime'),
-  location: attr('string'),
-  number: attr('number'),
-  numberRepresentation: attr('string'),
-  isFinal: attr('boolean'),
-  extraInfo: attr('string'),
-  releasedDocuments: attr('datetime'),
-  releasedDecisions: attr('datetime'),
-
-  agendas: hasMany('agenda', {
+export default class Meeting extends Model {
+  @service store;
+  @service intl;
+
+  @attr uri;
+  @attr('datetime') plannedStart;
+  @attr('datetime') startedOn;
+  @attr('datetime') endedOn;
+  @attr location;
+  @attr('number') number;
+  @attr numberRepresentation;
+  @attr('boolean') isFinal;
+  @attr extraInfo;
+  @attr('datetime') releasedDocuments;
+  @attr('datetime') releasedDecisions;
+
+  @hasMany('agenda', {
     inverse: null, serialize: false,
-  }),
-  requestedSubcases: hasMany('subcase'),
-  pieces: hasMany('piece'),
+  }) agendas;
+  @hasMany('subcase') requestedSubcases;
+  @hasMany('piece') pieces;
+  @hasMany('themis-publication-activity') themisPublicationActivities;
 
-  kind: belongsTo('concept'),
-  mainMeeting: belongsTo('meeting', {
+  @belongsTo('concept') kind;
+  @belongsTo('meeting', {
     inverse: null,
-  }),
-  newsletter: belongsTo('newsletter-info'),
-  mailCampaign: belongsTo('mail-campaign'),
-  agenda: belongsTo('agenda', {
+  }) mainMeeting;
+  @belongsTo('newsletter-info') newletter;
+  @belongsTo('mail-campaign') mailCampaign;
+  @belongsTo('agenda', {
     inverse: null,
-  }),
-
-  themisPublicationActivities: hasMany('themis-publication-activity'),
-
-  // This computed does not seem to be used anywhere
-  documentContainers: computed('pieces.@each.name', 'id', 'store', function() {
-    return PromiseArray.create({
-      promise: this.get('pieces').then((pieces) => {
-        if (pieces && pieces.get('length') > 0) {
-          return this.store.query('document-container', {
-            filter: {
-              pieces: {
-                meeting: {
-                  id: this.get('id'),
-                },
-              },
-            },
-            page: {
-              size: pieces.get('length'), // # documentContainers will always be <= # pieces
-            },
-            include: 'type,pieces,pieces.access-level,pieces.next-piece,pieces.previous-piece',
-          }).then((containers) => sortDocumentContainers(this.get('pieces'), containers));
-        }
-      }),
-    });
-  }),
-
-  canReleaseDecisions: computed('isFinal', 'releasedDecisions', function() {
-    return this.isFinal && !this.releasedDecisions;
-  }),
-
-  canReleaseDocuments: computed('isFinal', 'releasedDocuments', function() {
-    return this.isFinal && !this.releasedDocuments;
-  }),
-
-  latestAgenda: computed('agendas.[]', function() {
-    return PromiseObject.create({
-      promise: this.get('agendas').then((agendas) => {
-        const sortedAgendas = agendas.sortBy('agendaName').reverse();
-        return sortedAgendas.get('firstObject');
-      }),
-    });
-  }),
```

Comment:
This is still seems used in https://github.com/kanselarij-vlaanderen/frontend-kaleidos/blob/feature/KAS-3427-refactoren-en-verwijderen-van-computed-properties-op-ember-data-models/app/pods/newsletters/template.hbs#L75